### PR TITLE
Milestone: Feature performance audit

### PR DESCRIPTION
**Pull request by darikmohammed**
# Features

>  Optimize some slow queries in your database. 

## Added changes 

- Add an email column to your owners table.
- Add data to your database. 
- The following queries are taking too much time (1 sec = 1000ms can be considered as too much time for database query). 
  - `SELECT COUNT(*) FROM visits where animal_id = 4;`
  
Screen Shot 2022-08-16 at 3 31 23 PM

  - `SELECT * FROM visits where vet_id = 2;`
  
Screen Shot 2022-08-16 at 3 41 54 PM

  - `SELECT * FROM owners where email = 'owner_18327@mail.com';`
  
Screen Shot 2022-08-16 at 3 44 00 PM

- Find a way to improve the execution time of the queries.
  - `SELECT COUNT(*) FROM visits where animal_id = 4;`
  
Screen Shot 2022-08-16 at 3 40 30 PM

  - `SELECT * FROM visits where vet_id = 2;`
  
Screen Shot 2022-08-16 at 3 43 04 PM

  - `SELECT * FROM owners where email = 'owner_18327@mail.com';`
  
Screen Shot 2022-08-16 at 3 46 13 PM

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -75,4 +75,10 @@ CREATE TABLE VISITS (
...
     REFERENCES VETS(ID)
-); 
\ No newline at end of file
+); 
+
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+
+CREATE INDEX ON visists(animals_id);
+CREATE INDEX ON visists(vets_id);
+CREATE INDEX ON owners(email);
```
Comment (omar-muhamad) [inline schema.sql]: - [ ] Your indexing for `visits` and `owners` tables is working great like you shown in the screenshots you provided in the PR summary, But you have a typo error here in lines 82 and 83 It's supposed to be `visits` not `visists` so please fix that.

Other than that you have done a great job.

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @darikmohammed and @kiboma2021,

Nice job, you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Gitflow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [x] Please add your collaborator name and social links to `Authors` section in your README to make it more professional.

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: schema.sql (hunk 1 above)
Comment (darikmohammed) [reply schema.sql]: Thank you. Fixed

**Review by youmari — APPROVED** [review]
# Project Approved :trophy: :tada: 🟢 

Hi @darikmohammed @kiboma2021 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linters error.
:heavy_check_mark: Details PR title and description.
:heavy_check_mark: Good commit messages.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U02FTTCCU2V)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**